### PR TITLE
Misc mappings

### PR DESCRIPTION
**Pull request by maxpowa**
Little bit of this, little bit of that.

### CODE REVIEW

File: mappings/net/minecraft/server/ServerConfigurationManager.mapping [hunk 1]
```diff
@@ -29,11 +29,17 @@ CLASS none/mr net/minecraft/server/ServerConfigurationManager
 	FIELD l bannedIpList Lnone/mo;
 	FIELD m opList Lnone/ms;
 	FIELD n whitelist Lnone/my;
+	FIELD p playerSavehandler Lnone/bay;
```
Comment (asiekierka) [inline mappings/net/minecraft/server/ServerConfigurationManager.mapping]: playerSaveHandler

File: mappings/net/minecraft/server/config/BanInfo.mapping [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	FIELD d expiry Ljava/util/Date;
+	FIELD e reason Ljava/lang/String;
+	METHOD <init> (Ljava/lang/Object;Ljava/util/Date;Ljava/lang/String;Ljava/util/Date;Ljava/lang/String;)V
+		ARG 1 created
+		ARG 2 source
+		ARG 3 expiry
+		ARG 4 reason
+	METHOD a serialize (Lcom/google/gson/JsonObject;)V
+		ARG 0 json
+	METHOD c getBanExpiry ()Ljava/util/Date;
```
Comment (asiekierka) [inline mappings/net/minecraft/server/config/BanInfo.mapping]: getBanExpirationDate

File: mappings/net/minecraft/server/config/BanInfo.mapping [hunk 3]
```diff
@@ -0,0 +1,16 @@
+CLASS none/mm net/minecraft/server/config/BanInfo
+	FIELD a DATETIME_FORMAT Ljava/text/SimpleDateFormat;
+	FIELD b created Ljava/util/Date;
+	FIELD c source Ljava/lang/String;
+	FIELD d expiry Ljava/util/Date;
```
Comment (asiekierka) [inline mappings/net/minecraft/server/config/BanInfo.mapping]: expirationDate

File: mappings/net/minecraft/server/config/BanInfo.mapping [hunk 4]
```diff
@@ -0,0 +1,16 @@
+CLASS none/mm net/minecraft/server/config/BanInfo
+	FIELD a DATETIME_FORMAT Ljava/text/SimpleDateFormat;
+	FIELD b created Ljava/util/Date;
```
Comment (asiekierka) [inline mappings/net/minecraft/server/config/BanInfo.mapping]: creationDate

**Comment by maxpowa** [comment]
Done

**Comment by shadowfacts** [comment]
LGTM

**Pull request merged**